index.js

```diff
@@ -0,0 +1,201 @@
+import { ApolloServer, gql } from 'apollo-server-express';
+import { ApolloServerPluginDrainHttpServer } from 'apollo-server-core';
+import { GraphQLUpload, graphqlUploadExpress } from 'graphql-upload';
+import { MongoClient } from 'mongodb';
+import express from 'express';
+import http from 'http';
+import Dishes from './src/data-sources/Dishes.js';
+import Ingredients from './src/data-sources/Ingredients.js';
+import Shares from './src/data-sources/Shares.js';
+import Photos from './src/data-sources/Photos.js';
+
+const typeDefs = gql`
+    type Dish {
+        id: ID!
+        name: String!
+        meal: Meal!
+        ingredient: [RecipeIngredient]!
+        meat: Float
+        vegetable: Float
+        seafood: Float
+        spice: Boolean
+        cook_time: Int
+        iCook: String
+        amount: Int
+        solo(is: Boolean): Boolean
+        soup: Boolean
+        style: String
+    }
+
+    enum Meal {
+        lunch
+        dinner
+    }
+
+    type RecipeIngredient {
+        description: String!
```

Comment:
Let's use `ingredient: Ingredient!` so it can be linked to the ingredient instance.